contracts/LockedNORI.sol

```diff
@@ -163,8 +223,16 @@ contract LockedNORI is ERC777Upgradeable, ERC20PresetMinterPauserUpgradeable {
       emit UnvestedTokensRevoked(atTime, from, quantityRevoked);
       // TODO: destination address for clawed back tokens should be a role
```

Comment:
What do you think about this TODO?  Worth doing? I guess it's more flexible this way.  Maybe we should just burn the lNORI and `operatorSend` the actual NORI?